_vector-search/getting-started/auto-generated-embeddings.md

```diff
@@ -248,6 +253,70 @@ The response contains the matching documents:
+
+You can use automated workflows to create and deploy externally hosted models and create resources for various AI search types. In this example, you'll create the same search you've created using manual steps.
+
+### Step 1: Register and deploy the model
+
+To register and deploy a model, select the built-in workflow template for the model provider. For more information, see [Supported workflow templates]({{site.url}}{{site.baseurl}}/automating-configurations/workflow-templates/#supported-workflow-templates). Alternatively, to configure a custom model, use [Step 1 of the manual setup](#step-1-register-and-deploy-the-model).
+
+### Step 2: Configure a workflow
+
+Create and provision a semantic search workflow. You must provide the model ID for the model you configured. Review your selected workflow template [defaults](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/semantic-search-defaults.json) to determine whether you need to update any of the parameters. For example, if the model dimensionality is different from the default (`1024`), specify the dimensionality of your model in the `output_dimension` parameter. Change the workflow template default text field from `passage_text` to `text` in order to match the manual example:
+
+```json
+POST /_plugins/_flow_framework/workflow?use_case=semantic_search&provision=true
+{
+    "create_ingest_pipeline.model_id" : "mBGzipQB2gmRjlv_dOoB",
+    "text_embedding.field_map.output.dimension": "768",
+    "text_embedding.field_map.input": "text"
+}
+```
+{% include copy-curl.html %}
+
+OpenSearch responds with a workflow ID for the created workflow:
+
+```json
+{
+  "workflow_id" : "U_nMXJUBq_4FYQzMOS4B"
+}
+```
+
+To check the workflow status, send the following request:
+
+```json
+GET /_plugins/_flow_framework/workflow/U_nMXJUBq_4FYQzMOS4B/_status
+```
+{% include copy-curl.html %}
+
+Once the workflow completes, the `state` changes to `COMPLETED`. The workflow has created an ingest pipeline and an index called `my-nlp-index`:
+
+```json
+{
+  "workflow_id": "U_nMXJUBq_4FYQzMOS4B",
+  "state": "COMPLETED",
+  "resources_created": [
+    {
+      "workflow_step_id": "create_ingest_pipeline",
+      "workflow_step_name": "create_ingest_pipeline",
+      "resource_id": "nlp-ingest-pipeline",
+      "resource_type": "pipeline_id"
+    },
+    {
+      "workflow_step_name": "create_index",
+      "workflow_step_id": "create_index",
+      "resource_id": "my-nlp-index",
+      "resource_type": "index_name"
+    }
+  ]
+}
+```
+
+You can now continue with [Steps 4 and 5](#step-4-ingest-documents-into-the-index-ingest-documents-into-the-index) to ingest documents into the index and search the index.
```

Comment:
When it's plural, I think "steps" needs to be lowercase. Or you could separate them into Step 4 and Step 5.